pick.lic

```diff
@@ -708,48 +714,104 @@ class Pick
 
   def loot(box_noun)
     echo "Looting #{box_noun}..." if @debug
-    if DRC.bput("open my #{box_noun}", /You open/, /^In the .* you see .*\./, 'That is already open', 'It is locked') == 'It is locked'
+    
+    if DRC.bput("open my #{box_noun}", /^In the .* you see .*\./, 'That is already open', 'It is locked') == 'It is locked'
       echo 'Bug: Tried to loot locked box...'
       return
     end
-
-    if (@settings.fill_pouch_with_box || @settings.loot_specials.empty?)
+		
+    #pouch full, fill from lootpouch then try again      	
+    if Flags['full-lootpouch']
+      Flags.reset('full-lootpouch')
+      echo 'last match:, #{Regexp.last_match}'	 
+      echo 'last match0:, #{Regexp.last_match(0)}'	 
+      echo 'last match1:, #{Regexp.last_match(1)}'	       
```

Comment:
Were these echoes meant to be committed?

Reply:
removed